app/views/profiles/dashboard/_journey.html.erb

```diff
@@ -19,6 +19,13 @@
               <li> <i class="fa fa-money" aria-hidden="true"></i>  <%= t('.price') %>:€ <%= journey.car.price_hour.to_i %>/hr</li>
               <!-- <li> <i class="fa fa-clock-o" aria-hidden="true"></i>  <%#= t('.duration') %>:<%= journey.duration %> h </li> -->
               <li> <i class="fa fa-users" aria-hidden="true"></i  >  <%= t('.children') %>: <%= journey.remaining_seats %> </li>
+              
+              <% if journey.logs[0] %>
```

Comment:
avoid array access

also avoid business logic in views. `calculate_hours` could be the one checking `if journey.logs[0]` instead of making the view do that.